### PR TITLE
Xpp

### PR DESCRIPTION
**Pull request by Reniferowski**

### CODE REVIEW

File: quokka_editor_front/src/components/organisms/documentsEditor/DocumentsEditor.tsx [hunk 1]
```diff
@@ -131,29 +116,31 @@ const DocumentsEditor = () => {
...
-        <button
+      <div className="bg-project-theme-dark-350 px-8 py-4">
+        {/* <button
           className="rounded-full px-6 py-3 flex items-center justify-center bg-purple-500 text-white text-2xl absolute right-8 bottom-8 shadow-xl"
           type="button"
           onClick={() => getPDFHandler(id)}
         >
           Download PDF
-        </button>
+        </button> */}
```
Comment (Michal-Suhak) [inline quokka_editor_front/src/components/organisms/documentsEditor/DocumentsEditor.tsx:137]: Delete comments

File: quokka_editor_front/src/components/organisms/documentsEditor/EditorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      })
+      .then((res) => {
+        setClient({
+          ...client,
+          documentState: JSON.parse(res.data.content).join("\n"),
+          lastSyncedRevision: JSON.parse(res.data.last_revision),
+        });
+        setDocument(res.data);
+      });
+  }, []);
```
Comment (Michal-Suhak) [inline quokka_editor_front/src/components/organisms/documentsEditor/EditorPage.tsx:50]: Why are you still adding the requests and not extracting them to an external function?

File: quokka_editor_front/src/components/organisms/documentsEditor/ShareDocument.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+              <button
+                className="px-4 m-2 py-2 mx-8 rounded-full bg-gray-500 font-bold"
+                onClick={() => setShareModal(!shareModal)}
+              >
+                Cancel
+              </button>
+            </div>
+          </div>
+        </Modal>
+      ) : null}
```
Comment (Michal-Suhak) [inline quokka_editor_front/src/components/organisms/documentsEditor/ShareDocument.tsx:145]: Use `shareModal && ( rest of the code)` instead

File: quokka_editor_front/src/index.css [hunk 4]
```diff
@@ -14,11 +14,11 @@
...
-  background: #e7c8a0;
+  background: #5d5f6d;
   border-radius: 5px;
 }
 
 /* Handle on hover */
 ::-webkit-scrollbar-thumb:hover {
-  background: #e7c8a0;
+  background: #5d5f6d;
 }
```
Comment (Michal-Suhak) [inline quokka_editor_front/src/index.css:24]: You can extract bg color to css variable. Define the set of app colours and use them

File: quokka_editor_front/src/components/organisms/documentsEditor/EditorPage.tsx (hunk 2 above)
Comment (Reniferowski) [reply quokka_editor_front/src/components/organisms/documentsEditor/EditorPage.tsx:50]: i will do a new branch where I will move all the requests to a single file

**Review by Michal-Suhak — APPROVED** [review]

**Pull request merged**